.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: Continuous Integration 
+
+on:
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install dependencies
+        run: pip install -e ".[dev]"
+      - name: Run Tox
+        run: tox -e py
```

Comment:
What happens if the tests fail? Do the maintainers / codeowners get notified somehow?